instance/tasks.py

```diff
@@ -22,8 +22,9 @@
 
 # Imports #####################################################################
 
-from huey.djhuey import crontab, db_periodic_task, db_task
+import logging
 
+from huey.contrib.djhuey import crontab, db_periodic_task, db_task
```

Comment:
@antoviaque Nit: Any reason to list `huey` before `django` here (instead of sticking to alphabetical order)? Just wondering since both of them are third-party libraries.